index.js

```diff
@@ -752,7 +743,7 @@ class KeyringController extends EventEmitter {
   forgetKeyring(keyring) {
     if (keyring.forgetDevice) {
       keyring.forgetDevice();
-      this.persistAllKeyrings.bind(this)();
```

Comment:
This `.bind` was redundant, as we're calling `persistAllKeyrings` on `this` anyway. So it's a no-op.